nav2_graceful_motion_controller/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+cmake_minimum_required(VERSION 3.5)
+project(nav2_graceful_motion_controller)
+
+find_package(ament_cmake REQUIRED)
+find_package(nav2_common REQUIRED)
+find_package(nav2_core REQUIRED)
+find_package(nav2_costmap_2d REQUIRED)
+find_package(nav2_util REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(nav_msgs REQUIRED)
+find_package(pluginlib REQUIRED)
+find_package(tf2 REQUIRED)
+find_package(tf2_geometry_msgs REQUIRED)
+find_package(nav_2d_utils REQUIRED)
+find_package(angles REQUIRED)
+
+nav2_package()
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Does this need to specifically be C++17? `nav2-package()` already sets it to the REP2000 value